pyvrp/cpp/Individual.h

```diff
@@ -4,15 +4,61 @@
 #include "ProblemData.h"
 #include "XorShift128.h"
 
-#include <string>
+#include <iosfwd>
 #include <vector>
 
 class Individual
 {
     friend struct std::hash<Individual>;  // friend struct to enable hashing
 
     using Client = int;
-    using Route = std::vector<Client>;
+
+public:
+    /**
+     * A simple Route class that contains the route plan (as a vector), and
+     * some route statistics.
+     */
+    class Route
+    {
+        using Plan = std::vector<Client>;
+
+        Plan plan_ = {};         // Route plan (list of clients)
```

Comment:
I'm unsure about the name. We can probably do better. Maybe `tour`?

Reply:
- I find `plan` a bit vague, mostly because it's terminology I haven't heard before in the context of VRPs. First question that came to mind when reading this name was "a plan of what?" to which my answer was a plan of client visits.
	- How about simply renaming to `clients`? Does this add too much confusion?
	- Otherwise, what about `visits`? This also generalizes easily to more general vehicle routing problems (for example, pickups and deliveries).
- `tour` is OK with me as well for now but it implies that it always start/end at the same depot. If we ever allow for open VRPs/multiple depots then this might need to be changed again.

Reply:
Of these options, I think I like `visits` the most. I'll implement that now.